test/foundry-tests/MinimalBalance.t.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import {Test} from "forge-std/Test.sol";
+import {TokenFixture} from "./utils/TokenFixture.sol";
+import {MinimalBalance} from "../../contracts/MinimalBalance.sol";
+import {IMinimalBalance} from "../../contracts/interfaces/IMinimalBalance.sol";
+import {CurrencyLibrary, Currency} from "../../contracts/types/Currency.sol";
+
+contract MockMinimalBalanceImpl is MinimalBalance {
```

Comment:
nit: separate mock file generally preferred to multiple contracts in single file

Reply:
+1, in a different folder because its not a test file